queries/num.py

```diff
@@ -131,53 +59,101 @@ def convert(m):
+        "þrjár",
+        "fjórar",
+        "fimm",
+        "sex",
+        "sjö",
+        "átta",
+        "níu",
+        "tíu",
+        "ellefu",
+        "tólf",
+        "þrettán",
+        "fjórtán",
+        "fimmtán",
+        "sextán",
+        "sautján",
+        "átján",
+        "nítján",
+        "tuttugu",
+        "þrjátíu",
+        "fjörutíu",
+        "fimmtíu",
+        "sextíu",
+        "sjötíu",
+        "áttatíu",
+        "níutíu",
+    )
+)
 
-_NUM_NEUT_TO_KVK: Mapping[str, str] = {
-    "eitt": "ein",
-    "tvö": "tvær",
-    "þrjú": "þrjár",
-    "fjögur": "fjórar",
+_DeclensionMapping = Mapping[str, Mapping[str, Mapping[str, str]]]
+_NUM_NEUT_TO_DECL: _DeclensionMapping = {
+    "eitt": {
+        "kk": {"nf": "einn", "þf": "einn", "þgf": "einum", "ef": "eins"},
+        "kvk": {"nf": "ein", "þf": "eina", "þgf": "einni", "ef": "einnar"},
+        "hk": {"nf": "eitt", "þf": "eitt", "þgf": "einu", "ef": "eins"},
+    },
+    "tvö": {
+        "kk": {"nf": "tveir", "þf": "tvo", "þgf": "tveimur", "ef": "tveggja"},
+        "kvk": {"nf": "tvær", "þf": "tvær", "þgf": "tveimur", "ef": "tveggja"},
+        "hk": {"nf": "tvö", "þf": "tvö", "þgf": "tveimur", "ef": "tveggja"},
+    },
+    "þrjú": {
+        "kk": {"nf": "þrír", "þf": "þrjá", "þgf": "þremur", "ef": "þriggja"},
+        "kvk": {"nf": "þrjár", "þf": "þrjár", "þgf": "þremur", "ef": "þriggja"},
+        "hk": {"nf": "þrjú", "þf": "þrjú", "þgf": "þremur", "ef": "þriggja"},
+    },
+    "fjögur": {
+        "kk": {"nf": "fjórir", "þf": "fjóra", "þgf": "fjórum", "ef": "fjögurra"},
+        "kvk": {"nf": "fjórar", "þf": "fjórar", "þgf": "fjórum", "ef": "fjögurra"},
+        "hk": {"nf": "fjögur", "þf": "fjögur", "þgf": "fjórum", "ef": "fjögurra"},
+    },
 }
 
-_LARGE_NUMBERS: Tuple[Tuple[int, str], ...] = (
-    (int(1e21) * int(1e21) * int(1e6), "oktilljón"),  # 10^48
-    (int(1e21) * int(1e21), "septilljón"),  # 10^42
-    (int(1e21) * int(1e15), "sextilljón"),  # 10^36
-    (int(1e21) * int(1e9), "kvintilljón"),  # 10^30
-    (int(1e21) * int(1e6), "kvaðrilljarð"),  # 10^27
-    (int(1e21) * int(1e3), "kvaðrilljón"),  # 10^24
-    (int(1e21), "trilljarð"),  # 10^21
-    (int(1e18), "trilljón"),  # 10^18
-    (int(1e15), "billjarð"),  # 10^15
-    (int(1e12), "billjón"),  # 10^12
-    (int(1e9), "milljarð"),  # 10^9
-    (int(1e6), "milljón"),  # 10^6
+_LARGE_NUMBERS: Tuple[Tuple[int, str, str], ...] = (
+    (10 ** 48, "oktilljón", "kvk"),
+    (10 ** 42, "septilljón", "kvk"),
+    (10 ** 36, "sextilljón", "kvk"),
+    (10 ** 30, "kvintilljón", "kvk"),
+    (10 ** 27, "kvaðrilljarð", "kk"),
+    (10 ** 24, "kvaðrilljón", "kvk"),
+    (10 ** 21, "trilljarð", "kk"),
+    (10 ** 18, "trilljón", "kvk"),
+    (10 ** 15, "billjarð", "kk"),
+    (10 ** 12, "billjón", "kvk"),
+    (10 ** 9, "milljarð", "kk"),
+    (10 ** 6, "milljón", "kvk"),
 )
 
 
-def number_to_neutral(n: int = 0) -> str:
+def number_to_neutral(n: int = 0, *, eitt_hundrad: bool = False) -> str:
```

Comment:
The keyword arg eitt_hundrad is Icelandic, not English. Also, the function comment should explain what this arg flag does.

Reply:
Good point, fixed!